server/team-validator-async.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * Team Validator
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * Spawns a child process to validate teams.
+ *
+ * @license MIT
+ */
+
+import { crashlogger } from '../lib/crashlogger';
+
+declare var global: any;
+
+export class TeamValidatorAsync {
+	format: Format;
+
+	constructor(format: string) {
+		this.format = Dex.getFormat(format);
+	}
+
+	validateTeam(team: string, removeNicknames: boolean = false) {
+		let formatid = this.format.id;
+		if (this.format.customRules) formatid += '@@@' + this.format.customRules.join(',');
+		return PM.query({formatid, removeNicknames, team});
+	}
+
+	static get(format: string) {
+		return new TeamValidatorAsync(format);
+	}
+}
+
+export const get = TeamValidatorAsync.get;
+
+/*********************************************************
+ * Process manager
+ *********************************************************/
+
+import { QueryProcessManager } from '../lib/process-manager';
+
+// @ts-ignore
```

Comment:
Wait, what's being suppressed here?

Reply:
It was there in the original code. When I removed it, TSC didn't throw any errors.